drivers/syscon/efuse_bflb.c

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bflb_efuse
```

Comment:
```diff
-#define DT_DRV_COMPAT bflb_efuse
+#define DT_DRV_COMPAT bflb_sysconn_efuse
```

Reply:
I disagree on adding syscon to the driver binding name

Reply:
You should give a motive. There is no specific driver to efuse at moment and this API was selected.
When reading at devicetree will make easy to find where this driver is implemented and where is the binding of it. Besides, we still have the possibility to add a `bflb_sysconn` driver in future. Disagree by simple disagree not help to give a reason why not use as I proposed.

Reply:
You did not give a motive either. i find bflb,efuse more aesthetically pleasing and more descriptive ('syscon' is not descriptive of anything), adding the driver type name to the binding name is not common practice, and I doubt there will be a bflb_syscon driver in the future.
Additionaly, If that worked to find bindings, eeprom would be in a eeprom folder and not a folder named mtd for some reason.